EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,86 @@
+- [EIP-721](./eip-721.md)'s `ERC721Metadata` (`0x5b5e139f`)
+
+`EIP-XXXX` tokens _must not_ implement the interfaces:
+
+- [EIP-721](./eip-721.md)'s `ERC721` (`0x80ac58cd`)
+
+```solidity
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.6;
+
+/// @title Name-bound tokens
+/// @dev See https://eips.ethereum.org/EIPS/eip-XXXX
+///  Note: the ERC-165 identifier for this interface is 0x6352211e.
+interface IERCXXXX /* is ERC165, ERC721Metadata */ {
+  /// @dev This emits when ownership between ENS nodes of any NBT changes by any mechanism.
+  ///  This event emits when NBTs are created (`from` == 0x0) and destroyed
+  ///  (`to` == 0x0).
+  event Transfer(bytes32 indexed _from, bytes32 indexed _to, uint256 indexed _id);
+  /// @notice Find the owner of an NBT
+  /// @dev NBTs assigned to empty string namehash ('') are considered invalid, and queries
+  ///  about them do throw.
+  /// @param _tokenId The identifier for an NFT
+  /// @return The address of the owner of the NFT
+  function ownerOf(uint256 _tokenId) external view returns (bytes32);
+}
+```
+
+See [`EIP-721`](./eip-721.md) for a definition of its metadata JSON Schema.
+
+## Rationale
+
+### Interface
+
+`EIP-XXXX` shall be maximally backward-compatible but still only expose a minimal and simple to implement interface definition.
+
+As [`EIP-721`](./eip-721.md) tokens have seen widespread adoption with wallet providers and marketplaces, using its `ERC721Metadata` interface with [`EIP-165`](./eip-165.md) for feature-detection potentially allows implementers to support `EIP-XXXX` tokens out of the box.
+
+If an implementer of [`EIP-721`](./eip-721.md) properly built [`EIP-165`](./eip-165.md)'s `function supportsInterface(bytes4 interfaceID)` function, already by recognizing that [`EIP-721`](./eip-721.md)'s track and transfer interface component with the identifier `0x80ac58cd` is not implemented, transferring of a token should not be suggested as a user interface option.
+
+Still, however, since `EIP-XXXX` supports [`EIP-721`](./eip-721.md)'s `ERC721Metadata` extension, an account-bound token should be displayed in wallets and marketplaces with no changes needed.
+
+### Provenance Indexing
+
+NBTs can be indexed by tracking the emission of `event Transfer(bytes32 indexed _from, bytes32 indexed _to, uint256 indexed _id)`.
+
+We recommend implementers to validate `Transfer`'s `_from` field by comparing it to the transaction-level `_from` field to mitigate "sleepminting" attacks.
+
+## Backwards Compatibility
+
+We have adopted the [`EIP-165`](./eip-165.md) and `ERC721Metadata` functions purposefully to create a high degree of backward compatibility with [`EIP-721`](./eip-721.md). We have deliberately used [`EIP-721`](./eip-721.md) terminology such as `function ownerOf(...)` and `event Transfer` to minimize the effort of familiarization for `EIP-XXXX` implementers already familiar with, e.g., [`EIP-20`](./eip-20.md) or [`EIP-721`](./eip-721.md).
```

Comment:
This should be in the rationale, rather than the backwards compatibility section.